config.template.yaml

```diff
@@ -1,71 +1,87 @@
 ###############################################
 # Authorization using OAuth or Openid connect #
 ###############################################
+
 # ACCESS_TOKEN_SECRET_KEY should contain a random string with enough entropy (at least 32 bytes long) to securely sign all access_tokens for OAuth and Openid connect
-ACCESS_TOKEN_SECRET_KEY: ""
+ACCESS_TOKEN_SECRET_KEY: #YWZOHliiI53lJMJc5BI_WbGbA4GF2T7Wbt1airIhOXEa3c021c4-1c55-4182-b141-7778bcc8fac4
```

Comment:
I'm not sure how this value was generated, but I wonder if someone could have concatenated a random sequence with an uuid. I suggest using a value that is really a random sequence, as one could argue that uuid may not be suitable to use as secret.